include/z64actor.h

```diff
@@ -254,6 +254,9 @@ if neither of the above are set : blue
 0x2000 : translucent, else opaque
 */
 
+#define DYNA_TRANSFORM_POS (1 << 0) // Position of the actors on top follows the dynapoly actor's movement.
+#define DYNA_TRANSFORM_ROT_Y (1 << 1) // The Y rotation of the actors on top follows the dynapoly actor's Y rotation.
```

Comment:
Maybe I'm a bit biased for spending too much time in Camera, but would this be better just as `DYNA_TRANSFORM_YAW`, or would we prefer to keep it to `ROT_Y` when possible?

Reply:
My motivation for rotY over yaw is to see yaw as always an angle around the upwards vertical axis, and rotY as an angle around +y, which is not necessarily upwards vertical (typically if rotX or rotZ is non-zero)

The implementation for this flag just changes rot.y directly, it doesn't account for x/z rotations, like I'd expect it would if it updated the "yaw"

but that's just my way of making a distinction between rotY and yaw, and typically it doesn't matter which to use. Yaw has been suggested before and I'm okay with changing it if that's what the fans demand